### PR TITLE
Update our docs to document the 0.1.1 release.

### PR DESCRIPTION
**Pull request by mattmoor**

### CODE REVIEW

File: install/Knative-with-AKS.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ Knative depends on Istio.
 
 1. Install Istio:
     ```bash
-    kubectl apply -f https://raw.githubusercontent.com/knative/serving/v0.1.0/third_party/istio-0.8.0/istio.yaml
+    kubectl apply -f https://raw.githubusercontent.com/knative/serving/v0.1.1/third_party/istio-0.8.0/istio.yaml
```
Comment (evankanderson) [inline install/Knative-with-AKS.md]: Do we still want to install the istio 0.8.0 release?

File: install/Knative-with-GKE.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ Knative depends on Istio.
 
 1. Install Istio:
     ```bash
-    kubectl apply -f https://raw.githubusercontent.com/knative/serving/v0.1.0/third_party/istio-0.8.0/istio.yaml
+    kubectl apply -f https://raw.githubusercontent.com/knative/serving/v0.1.1/third_party/istio-0.8.0/istio.yaml
```
Comment (evankanderson) [inline install/Knative-with-GKE.md]: Same

File: install/Knative-with-Gardener.md [hunk 3]
```diff
@@ -159,10 +159,10 @@ spec:
 And of course create the respectve `ConfigMaps`:
 
 ```
-curl https://raw.githubusercontent.com/knative/serving/v0.1.0/third_party/istio-0.8.0/istio.yaml
+curl https://raw.githubusercontent.com/knative/serving/v0.1.1/third_party/istio-0.8.0/istio.yaml
```
Comment (evankanderson) [inline install/Knative-with-Gardener.md]: Here's an extra one.

File: install/Knative-with-IKS.md [hunk 4]
```diff
@@ -167,7 +167,7 @@ You can install the Knative Serving and Build components together, or Build on i
 1. Run the `kubectl apply` command to install
    [Knative Build](https://github.com/knative/build) and its dependencies:
     ```bash
-    kubectl apply -f https://raw.githubusercontent.com/knative/serving/v0.1.0/third_party/config/build/release.yaml
+    kubectl apply -f https://raw.githubusercontent.com/knative/serving/v0.1.1/third_party/config/build/release.yaml
```
Comment (evankanderson) [inline install/Knative-with-IKS.md]: Should this point to the vendored serving copy, or the separate build release (if it exists)?

File: install/Knative-with-AKS.md (hunk 1 above)
Comment (mattmoor) [reply install/Knative-with-AKS.md]: Yes.  We're compatible with 1.0, but only 0.8.0 is in the release branch

File: install/Knative-with-IKS.md (hunk 4 above)
Comment (mattmoor) [reply install/Knative-with-IKS.md]: I think I'd rather make more substantive changes in a broader update around decoupling, if that's alright?

**Pull request merged**